cmd/prysmctl/db/span.go

```diff
@@ -0,0 +1,287 @@
+	"github.com/urfave/cli/v2"
+)
+
+const DefaultChunkKind = types.MinSpan
+
+var (
+	f = struct {
+		Path                          string
+		ValidatorIndex                uint64
+		Epoch                         uint64
+		ChunkKind                     string
+		ChunkSize                     uint64
+		ValidatorChunkSize            uint64
+		HistoryLength                 uint64
+		IsDisplayAllValidatorsInChunk bool
+		IsDisplayAllEpochsInChunk     bool
+	}{}
+
+	slasherDefaultParams = slasher.DefaultParams()
+)
+
+var spanCmd = &cli.Command{
+	Name:  "span",
+	Usage: "visualise values in db span bucket",
+	Action: func(c *cli.Context) error {
+		if err := spanAction(c); err != nil {
+			return errors.Wrapf(err, "visualise values in db span bucket failed")
+		}
+		return nil
+	},
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:        "db_path_directory",
+			Usage:       "path to directory containing beaconchain.db",
+			Destination: &f.Path,
+			Required:    true,
+		},
+		&cli.Uint64Flag{
+			Name:        "validator_index",
+			Usage:       "filter by validator index",
+			Destination: &f.ValidatorIndex,
+			Required:    true,
+		},
+		&cli.Uint64Flag{
+			Name:        "epoch",
+			Usage:       "filter by epoch",
+			Destination: &f.Epoch,
+			Required:    true,
+		},
+		&cli.StringFlag{
+			Name:        "chunk_kind",
+			Usage:       "chunk kind to query (maxspan|minspan)",
+			Destination: &f.ChunkKind,
+			Value:       DefaultChunkKind.String(),
+			DefaultText: DefaultChunkKind.String(),
+		},
+		&cli.Uint64Flag{
+			Name:        "chunk_size",
+			Usage:       "chunk size to query",
+			Destination: &f.ChunkSize,
+			DefaultText: fmt.Sprintf("%d", slasherDefaultParams.ChunkSize()),
+		},
+		&cli.Uint64Flag{
+			Name:        "validator_chunk_size",
+			Usage:       "validator chunk size to query",
+			Destination: &f.ValidatorChunkSize,
+			DefaultText: fmt.Sprintf("%d", slasherDefaultParams.ValidatorChunkSize()),
+		},
+		&cli.Uint64Flag{
+			Name:        "history_length",
+			Usage:       "history length to query",
+			Destination: &f.HistoryLength,
+			DefaultText: fmt.Sprintf("%d", slasherDefaultParams.HistoryLength()),
+		},
+		&cli.BoolFlag{
+			Name:        "display_all_validators_in_chunk",
+			Usage:       "display all validators in chunk",
+			Destination: &f.IsDisplayAllValidatorsInChunk,
+		},
+		&cli.BoolFlag{
+			Name:        "display_all_epochs_in_chunk",
+			Usage:       "display all epochs in chunk",
+			Destination: &f.IsDisplayAllEpochsInChunk,
+		},
+	},
+}
+
+func spanAction(cliCtx *cli.Context) error {
+	var (
+		chunk             slasher.Chunker
+		validatorChunkIdx uint64
+
+		err error
+	)
+
+	// context
+	ctx := cliCtx.Context
+
+	// variables
+	chunkKind := getChunkKind()
+	params := getSlasherParams()
+	i := primitives.ValidatorIndex(f.ValidatorIndex)
+	epoch := primitives.Epoch(f.Epoch)
+
+	// display configuration
+	fmt.Printf("############################# CONFIGURATION ################################\n")
```

Comment:
Maybe `CONFIGURATION` could be replaced by `SLASHER PARAMS` and `# Slasher Params` could be removed.